src/templates/globals/wrapper/index.tsx

```diff
@@ -40,8 +41,10 @@ export function Wrapper({ bannerData, children }: LayoutProps) {
 
   return (
     <>
-      {/* {showBanners ? mapBanners : null} */}
+      {showBanners ? mapBanners : null}
```

Comment:
Will you move these banners below the header?
looking at https://pr141-yb1aiq9d4a369sojvmry5r2rman1gavn.tugboat.vfs.va.gov/butler-health-care/stories/ vs https://www.va.gov/butler-health-care/stories/

Reply:
good call, will fix